WrightTools/data/_spcm.py

```diff
@@ -74,27 +70,54 @@ def from_spcm(filepath, name=None, *, delimiter=",", format=None, parent=None, v
                     headers["resolution"] = int(value.strip(" bits ADC"))
                 else:
                     headers[key.strip()] = value.strip()
+        line = f.readline().strip()
+        while "_BEGIN" in line:
+            header_lines += 1
+            section = line.split("_BEGIN")[0]
+            while True:
+                line = f.readline().strip()
+                header_lines += 1
+                if section + "_END" in line:
+                    break
+                if section == "SYS_PARA":
+                    use_type = {
+                        "B": lambda b: int(b) == 1,
+                        "C": str,  # e.g. #SP [SP_OVERFL,C,N]
+                        "F": float,
+                        "I": int,
+                        "L": int,  # e.g. #DI [DI_MAXCNT,L,128]
+                        "S": str,
+                        "U": int,  # unsigned int?
+                    }
+                    item = line[line.find("[") + 1 : line.find("]")].split(",")
+                    key = item[0]
+                    value = use_type[item[1]](item[2])
+                    headers[key] = value
+                else:
+                    splitted = line.split()
+                    value = splitted[-1][1:-1].split(",")
+                    key = " ".join(splitted[:-1])
+                    headers[key] = value
+            line = f.readline().strip()
+            if "END" in line:
+                header_lines += 1
+                break
+    if "Date" in headers.keys() and "Time" in headers.keys():
+        # NOTE:  reports created in local time, no-way to calculate absolute time
+        created = " ".join([headers["Date"], headers["Time"]])
+        created = time.strptime(created, "%Y-%m-%d %H:%M:%S")  
+        created = timestamp.TimeStamp(time.mktime(created)).RFC3339
```

Comment:
You get the created time here, but looks like you never do anything with it should be added to the kwargs as the "special/reserved" attribute `"created"`